firewall/privacy_mapper.go

```diff
@@ -845,3 +858,140 @@ func CryptoRandIntn(n int) (int, error) {
 
 	return int(nBig.Int64()), nil
 }
+
+// ObfuscateConfig alters the config string by replacing sensitive data with
+// random values and returns new replacement pairs. We only substitute items in
+// strings, numbers are left unchanged.
+func ObfuscateConfig(db firewalldb.PrivacyMapReader, configB []byte) ([]byte,
+	map[string]string, error) {
+
+	if len(configB) == 0 {
+		return nil, nil, nil
+	}
+
+	// We assume that the config is a json dict.
+	var configMap map[string]any
+	err := json.Unmarshal(configB, &configMap)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	privMapPairs := make(map[string]string)
+	newConfigMap := make(map[string]any)
+	for k, v := range configMap {
+		// We only substitute items in lists.
+		list, ok := v.([]any)
+		if !ok {
+			newConfigMap[k] = v
+			continue
+		}
+
+		// We only substitute items in lists of strings.
+		stringList := make([]string, len(list))
+		anyString := false
+		allStrings := true
+		for i, item := range list {
+			item, ok := item.(string)
+			allStrings = allStrings && ok
+			anyString = anyString || ok
+			stringList[i] = item
```

Comment:
ah ok it doesnt panic since `item` at this point is empty. But maybe just for clarity, let's do:
```
if !ok {
    continue
}

stringList[i] = item
```